packages/playground/src/components/pull-request/pull-request-changes.tsx

```diff
@@ -111,7 +119,7 @@ export function PullRequestChanges({ data }: DataProps) {
   return (
     <div className="flex flex-col gap-4">
       {data.map((item, index) => (
-        <PullRequestAccordion key={index} header={item} data={item?.data} />
+        <PullRequestAccordion key={item?.title && `${item?.title}-${index}`} header={item} data={item?.data} />
```

Comment:
I should have been more I meant was to just have `index` as key in case title is `empty/undefined`.

If `item?.title` is empty then it would mean empty or no `key`, React would then start complaining for duplication.

Just `item?.title ? ${item?.title}-${index} : ${index}` should be enough.

Reply:
done